src/actions/data.js

```diff
@@ -0,0 +1,23 @@
+const { ActionTransport } = require('@microfleet/core');
+const pick = require('lodash/pick');
+
+const { FILES_DATA } = require('../constant');
+const fetchData = require('../utils/fetch-data');
+
+/**
+ * File information
+ * @param  {Object} params.fileId
+ * @param  {Object} [params.fields]
+ * @return {Promise}
+ */
+async function getFileData({ params }) {
+  const { fileId, fields } = params;
+  const fieldsToQuery = ['uploadId', ...fields];
+
+  const file = await fetchData.call(this, `${FILES_DATA}:${fileId}`);
```

Comment:
https://github.com/makeomatic/ms-files/blob/master/src/utils/fetch-data.js#L102

omitFields is allowed by the script. Its best to alter lua script and add new "pickFields"
otherwise there is a lot of work that need not be done - picking on app level is much slower than on the database side